driver.sh

```diff
@@ -70,6 +70,17 @@ setup_variables() {
       qemu_cmdline=( -drive "file=images/x86_64/rootfs.ext4,format=raw,if=ide"
                      -append "console=ttyS0 root=/dev/sda" ) ;;
 
+    "ppc64le")
+      config=powernv_defconfig
+      image_name=zImage.epapr
+      qemu="qemu-system-ppc64"
+      qemu_cmdline=( -M powernv -m 3G -nographic
```

Comment:
Does the `-m 3G` here conflict with the one one L162? (how much memory is allotted to the VM if you specify `-m` twice?)  Can you boot in `-m 512`?

Reply:
If not, I suggest adding `-m 512` to the other case statements, and removing the default from L162.

Reply:
The entry that comes last takes precedence. This is why I moved the `${qemu_cmdline[@]}` to after `-m 512` in `boot_qemu()`.

We can't boot with less due to some limitations (I think in Qemu? I'm chasing it up with some people who know the details).

I'll make it explicit for all entries.

Reply:
2G works for me when booting a powernv image.

Reply:
qemu-system-ppc64 -M powernv -cpu POWER8 -m 2G -kernel arch/powerpc/boot/zImage.epapr -initrd /var/cache/buildbot/ppc64/rootfs-el.cpio -nographic -vga none -monitor null -no-reboot --append 'rdinit=/sbin/init panic=-1 console=tty console=hvc0'

1G crashes with a NULL pointer access in unpack_to_rootfs().

Reply:
@groeck Thanks. I will sort out that crash and we can look at reducing the amount of ram in the future.